az-core/src/main/java/azkaban/Constants.java

```diff
@@ -670,6 +670,10 @@
         KUBERNETES_FLOW_CONTAINER_PREFIX + "max.allowed.memory";
     public static final String KUBERNETES_FLOW_CONTAINER_MEMORY_REQUEST =
         KUBERNETES_FLOW_CONTAINER_PREFIX + "memory.request";
+    public static final String KUBERNETES_FLOW_CONTAINER_DISK_REQUEST =
+        KUBERNETES_FLOW_CONTAINER_PREFIX + "disk.request";
+    public static final String KUBERNETES_FLOW_CONTAINER_MAX_ALLOWED_DISK =
+        KUBERNETES_FLOW_CONTAINER_PREFIX + "max.allowed.disk";
```

Comment:
Wondering if we should refer to ephemeral storage specifically in the name of the property to avoid any confusion.

Reply:
I had the same question about the `flow.container.disk.request` param. I believe users know that this disk is a temporary space.

Reply:
From flow container's  perspective that is the disk it has.

Reply:
Besides, its a short name. let me know if you want me to change it.

Reply:
let's keep current name then, thanks for clarifying!